azavea/raster-foundry
# Add day-of-month filter to backend

## Overview

This PR adds filtering by day-of-month to the backend.

The query parameters for this are `minDayOfMonth` and `maxDayOfMonth`.

### Checklist


## Testing Instructions

 * Make sure your app-server is restarted
 * Curl to test various combinations of `minDayOfMonth` and `maxDayOfMonth` like so:
```
curl -H "Content-Type: application/json" \
     -H "Authorization: Bearer <TOKEN>" \
     http://localhost:9000/api/scenes?minDayOfMonth=15&maxDayOfMonth=30&page=0&pageSize=10&sort=createdAt,desc
```
